MediaBrowser.Providers/Plugins/AudioDb/AudioDbAlbumProvider.cs

```diff
@@ -20,6 +20,9 @@
 
 namespace MediaBrowser.Providers.Plugins.AudioDb
 {
+    /// <summary>
+    /// TheAdioDb album provider.
```

Comment:
```suggestion
    /// TheAudioDb album provider.
```

Reply:
Good catch. I seem to have a lot of trouble with the word audio 😅